### DISCUSSION TITLE
Tutorial for modification?

### DISCUSSION THREAD

**Discussion by fl-yu** [alshedivat/al-folio#607] [Q&A] [answered]
Thank you so much for this awesome template!! I am wondering if there is a tutorial or guidance page for customized modification?

**Comment by mbarbetti** [comment, marked as answer]
The first steps to follow are reported in the [`README.md`](https://github.com/alshedivat/al-folio/blob/master/README.md). 
You can start from there!

**Comment by ghost** [comment]
I wish there was one too
